src/main/java/io/appform/dropwizard/sharding/sharding/ConsistentHashBucketIdExtractor.java

```diff
@@ -15,20 +15,18 @@
  *
  */
 
-package io.appform.dropwizard.sharding.sharding.impl;
+package io.appform.dropwizard.sharding.sharding;
 
 import com.google.common.hash.Hashing;
 import io.appform.dropwizard.sharding.DBShardingBundleBase;
-import io.appform.dropwizard.sharding.sharding.BucketIdExtractor;
-import io.appform.dropwizard.sharding.sharding.ShardManager;
 
 import java.nio.charset.StandardCharsets;
 import java.util.Map;
 
 /**
  * Generates bucket id on the basis of murmur128 of the key.
  */
-public class ConsistentHashBucketIdExtractor<T> implements BucketIdExtractor<T> {
```

Comment:
We can keep the interface internally in the bundle and not expose to outside world

Reply:
ack. But can we go ahead with the removal of other implementations - RandomBucketIdExtractor and MultiTenantConsistentHashBucketIdExtractor to avoid confusion ?

Reply:
closing this. raised a new MR.